Makefile

```diff
@@ -118,6 +118,9 @@ gosec: ## Run gosec locally
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet test-fast ## Run tests.
 
+integration:
```

Comment:
WDYT about making it obvious that this target runs tests (for example `integration-test`)?